### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ninsch3000**
Adjust repo directory structure and output/logdir structure options to match the [TPAS repo](https://github.com/zavolanlab/tandem-pas).
Other small fixes, mainly docs updates.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,23 +1,335 @@
...
+# -*- mode: gitignore; -*-
 *~
-# bam files
-PAQR/data/bam_files
-# data from PAQR test runs
-PAQR/log_output.log
-PAQR/HNRNPC_KD/
-PAQR/data/samplewise_TIN/
-PAQR/.snakemake/
-PAQR/scripts/deNovo-used-sites-and-usage-inference.testVersion.py
```
Comment (AngryMaciek) [inline .gitignore:12]: Some od these look like leftovers from the old, original PAQR repo, should be removed?

File: .gitignore [hunk 2]
```diff
@@ -1,23 +1,335 @@
...
-KAPAC/doIt.sh
-KAPAC/RESULTS/
-KAPAC/RESULTS_FORMER.log
-KAPAC/RESULTS_FORMER/
-KAPAC/RESULTS_FORMER_AFTER_CLEANUP/
-KAPAC/RESULTS.log
-# Rhistory
-*.Rhistory
-KAPAC/DATA/kmer_counts_all.tsv
```
Comment (AngryMaciek) [inline .gitignore:21]: These are definitely leftovers, should be removed.

File: README.md [hunk 3]
```diff
@@ -77,10 +81,19 @@ The file `configs/config.yaml` contains all information about used parameter val
...
 
 ## Execution
-Create a new directory for your analysis within this directory and cd into it. Make sure you have the conda environment `paqr2` activated. For your convenience, the directory `execution` contains bash scripts that can be used to start local and slurm runs, using either singularity or conda. In order for the latter to work, you will have to specify the partition to be used in the respective bash scripts at `-p [PARTITION]`.
+Create a new directory for your analysis within this directory and cd into it. Make sure you have the conda environment `paqr2` activated. For your convenience, the directory `execute` contains bash scripts that can be used to start local and slurm runs, using either singularity or conda.
+
+For example, you could run the example config `configs/config.yml` locally with singularity with:
+
+```bash
+bash snakemake_local_run_singularity_containers.sh configs/config.yml
+```
```
Comment (AngryMaciek) [inline README.md:96]: We can leave it like this for now but I want to give you a short heads-up.
Last week I have updated my PAQR version I have in MAPP such that I have added snakemake profiles and now there is just one main bash script to call the pipeline. Depending on the command line arguments to that `run.sh` script a different snakemake profile will be run. Once MAPP is public I can upgrade this version as well.
But FYI, we are asynchronous again. I needed to incorporate these recent changes in order to prepare MAPP to be easily run by the reviewers.

File: execute/snakemake_cluster_run_conda_environments.sh [hunk 4]
```diff
@@ -2,11 +2,12 @@
 
 # Run the pipeline on a computational cluster
 # with conda virtual environments
+# Usage: bash snakemake_cluster_run_conda_environments.sh ../configs/config.yaml
 
 cleanup () {
     rc=$?
     # rm -rf .snakemake/
-    rm -rf ../output/
+    # rm -rf ../output/
```
Comment (AngryMaciek) [inline execute/snakemake_cluster_run_conda_environments.sh:10]: good catch, we don't want to clense everything on error...

**Review by AngryMaciek — COMMENTED** [review]
Looks OK to me!

image

File: .gitignore (hunk 2 above)
Comment (ninsch3000) [reply .gitignore:21]: yeah... red color means they have been removed, right?

Comment (AngryMaciek) [reply .gitignore:21]: Oh, sorry, that was stupid of me....

**Review by AngryMaciek — APPROVED** [review]

**Pull request merged**